src/MCPClient/lib/clientScripts/failedSIPCleanup.py

```diff
@@ -22,9 +22,10 @@ def main(fail_type, sip_uuid):
     models.SIPArrange.objects.filter(sip_id=sip_uuid).delete()
 
     # Update storage service that reingest failed
-    api = storage_service._storage_api()
+    session = storage_service._storage_api_session()
+    url = storage_service._storage_service_url() + 'file/' + sip_uuid + '/'
```

Comment:
Do you know why we don't have a function in `storageService.py` for this case? It feels clumsy that we're using functions that are meant to be used only internally.